Simplenote/SPTracker+Extensions.swift

```diff
@@ -29,6 +29,41 @@ extension SPTracker {
     }
 }
 
+// MARK: - In App Notifications
+//
+extension SPTracker {
+
+    static func trackPresentedNotice(kind: NoticeType, additionalProperties: [String: String]? = nil) {
+        var properties = ["notice": kind.rawValue]
+
+        if let additionalProperties = additionalProperties {
+            for key in additionalProperties.keys {
+                properties[key] = additionalProperties[key]
+            }
+        }
+        trackAutomatticEvent(withName: "notice_presented", properties: properties)
+    }
+
+    static func trackPreformedNoticeAction(kind: NoticeType, action: NoticeActionType) {
```

Comment:
```suggestion
    static func trackPreformedNoticeAction(ofType actionType: NoticeActionType, noticeType: NoticeType) {
```

Reply:
Or something like that! I'm leaning towards `Type` because both enums read `NoticeXXXType`